src/Contrib/Service/Masa.Contrib.Service.MinimalAPIs/Url.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Contrib.Service.MinimalAPIs;
+
+public class Url
+{
+    /// <summary>
+    /// The prefix, the default is null, consistent with the global prefix, if it is string.Empty, the Prefix is ignored
```

Comment:
Formatter is `$"{Prefix}/{Version}/{ServiceName}"`, any one `IsNullOrWhiteSpace` would be ignored.